viewer-prototype/src/browser/trace-viewer/components/timegraph-output-component.tsx

```diff
@@ -66,6 +66,10 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
         };
         this.chartLayer = new TimeGraphChart('timeGraphChart', providers, this.rowController);
         this.vscrollLayer = new TimeGraphVerticalScrollbar('timeGraphVerticalScrollbar', this.rowController);
+        
+        this.rowController.onVerticalOffsetChangedHandler((offset)=>{ 
+            this.treeRef.current.scrollTop=this.rowController.verticalOffset;
```

Comment:
Is this.rowController.verticalOffset == offset? If so, use the offset in parameter. Otherwise, what is the offset parameter?